content/_code-samples/airgapped-wallet/py/airgapped-wallet-TEST.py

```diff
@@ -121,7 +121,7 @@ def create_wallet_directory():
         print("- OS Detected: Windows")
         File = PureWindowsPath(str(usr) + '/WalletTEST')
         Path_ = str(PureWindowsPath(str(usr)))
-    if OS == "Linux":
+    else:
         print("- OS Detected: Linux")
```

Comment:
I guess this output is no longer accurate with the else statement, but it's no big deal